devel/ci-run-e2e.sh

```diff
@@ -52,6 +52,15 @@ if [[ "$IS_OPENSHIFT" == "true" ]] ; then
   export SERVICE_IP_PREFIX="172.30.0"
 fi
 
+# When running in our CI environment the Docker network's subnet choice will cause issues with routing
+# This works this around till we have a way to properly patch this.
+docker network create --driver=bridge --subnet=192.168.0.0/16 --gateway 192.168.0.1 kind
+# Wait a second for the network to be created so kind does not overwrite it
+sleep 5
```

Comment:
The comment here says 1 second but we actually sleep for 5 🙂

Is this wait really required? 🙁 can we replace it with a loop that waits for `docker network list` or whatever equivalent to finish?

Also, if the network already exists (ie after an existing kind cluster has been created already, or a user running e2e tests twice), this will cause multiple networks with the same name to exist, causing https://github.com/kubernetes-sigs/kind/issues/1596 right?

Reply:
Good point, will look into a better check

Reply:
So docker inspect throws a nice error when it does not exist, used that to do checks